README.md

```diff
@@ -1,75 +1,29 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-> Description the project.
```

Comment:
- Please, kindly add some description about your project so that users can have an idea what the project is all about as this would make your readme more professional.